modules/bitgo/example/js/btc/create-lightning-invoice.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Create a lightning invoice.
+ *
+ * Copyright 2022 BitGo, Inc.  All Rights Reserved.
+ */
+const BitGoJS = require('bitgo');
+
+// change this to env: 'production' when you are ready for production
+const env = 'test';
+// change coin to 'btc' when working with production
+const coin = 'tbtc';
+
+const bitgo = new BitGoJS.BitGo({ env });
+
+// set your access token here
+const accessToken = '';
+
+// set your wallet from the YYYYY parameter here in the URL on app.bitgo-test.com
+// https://test.bitgo.com/enterprise/XXXXXXXXX/coin/tbtc/YYYYY/transactions
+const walletId = '';
+
+async function createLightningInvoice() {
+  bitgo.authenticateWithAccessToken({ accessToken });
+  const wallet = await bitgo.coin(coin).wallets().get({ id: walletId });
+  const lightning = wallet.lightning();
+
+  const walletInvoice = await lightning.createInvoice({ value: 100000, memo: 'Test Payment' });
+  console.log('Wallet Invoice:', walletInvoice.invoice);
```

Comment:
I think rather than "Wallet Invoice" it'd be more accurate/specific to call this a "Lightning Invoice"